UrbanAirship.iOS.Core.nuspec

```diff
@@ -1,16 +1,16 @@
 <?xml version="1.0" encoding="utf-8"?>
```

Comment:
I think we still want a UrbanAirship.iOS.nuspec that depends on Core, Automation, MessageCenter, and ExtendedActions

Reply:
OK.